posthog/models/cohort/util.py

```diff
@@ -324,6 +325,7 @@ def recalculate_cohortpeople(
             "new_version": pending_version,
         },
         settings={"optimize_on_insert": 0},
+        workload=Workload.OFFLINE,
```

Comment:
Alternatively, this could also just drop the `kind="cohort_calculation"` bit from above and I _think_ that'd fall through to the `celery` default for the task and use the default from #22493. I think a reasonable argument could be made for either approach here?